static/app/views/releases/list/index.tsx

```diff
@@ -325,16 +334,21 @@ class ReleasesList extends AsyncView<Props, State> {
   }
 
   renderHealthCta() {
-    const {selection, organization} = this.props;
+    const {organization} = this.props;
     const {hasSessions, releases} = this.state;
 
-    const selectedProjectId =
-      selection.projects && selection.projects.length === 1 && selection.projects[0];
-    const selectedProject = organization.projects?.find(
-      p => p.id === `${selectedProjectId}`
-    );
-
-    if (!selectedProject || hasSessions !== false || !releases?.length) {
+    const selectedProject = this.getSelectedProject();
+    const isMobileProject =
+      selectedProject &&
+      selectedProject.platform &&
+      ([...mobile, ...desktop] as string[]).includes(selectedProject.platform as string);
+
+    if (
+      !selectedProject ||
+      hasSessions !== false ||
+      !releases?.length ||
+      !isMobileProject
+    ) {
```

Comment:
@taylangocmen This health banner should be visible to everyone, not just mobile+desktop, can you please change that back?

Reply:
👍